cobalt/browser/cobalt_secure_navigation_throttle.cc

```diff
@@ -95,8 +94,8 @@ CobaltSecureNavigationThrottle::EnforceCSPHeaders() {
         content::NavigationThrottle::CANCEL, net::ERR_BLOCKED_BY_CLIENT);
   }
 
-  if (response_headers->GetNormalizedHeader("Content-Security-Policy",
-                                            &CSP_value)) {
+  if (std::optional<std::string> CSP_value = 
+        response_headers->GetNormalizedHeader("Content-Security-Policy")) {
```

Comment:
I assume this function before returned true on success and now returns a value for the std::optional on success?

It doesn't seem like we need a variable CSP_value anymore though, and can just check `if (response_headers->GetNormalizedHeader(...))` (I don't see CSP_value used below)